submitted_models/costar_shafter_sensor_config_1/specifications.md

```diff
@@ -0,0 +1,77 @@
+<!-- This is a Markdown description of a robot model submitted for inclusion in the
+DARPA Subterranean Challenge Technology Repository -->
+
+# COSTAR Shafter Sensor Config 1
+<div style="text-align: justify">
```

Comment:
I would recommend removing all of the `<div>` tags from this file, and any other markdown files that may have `<div>` tags in them. We didn't require `<div>` tags from other model submissions in their files, and by including `<div>` tags, it seems to change the markdown rendering behavior when viewing the pages in GitHub (for example, in the `Motion Characteristics` section, the 'model preparation guide' hyperlink does not show up when using the `<div>` tag).

I removed a few `<div>` tags from this file and tested the rendering locally - it seems to fix a lot of the markdown rendering issues and also doesn't ruin any of the text alignment.